src/controllers/userController.js

```diff
@@ -24,11 +26,31 @@ exports.signupUser = async (req, res) => {
             // console.log('Admin claim set for user:', userRecord.uid);
         }
 
+        // URL of the default profile picture served by your server
+        const defaultProfilePicture = 'http://localhost:8000/images/userProfileImage.jpg';
+        const defaultProfilePicture1 = 'gs://learninghub-ggpacteam3.appspot.com/images/userProfileImage.jpg';
+        //gs://learninghub-ggpacteam3.appspot.com/images/userProfileImage.jpg
```

Comment:
If we aren't using code, we should delete it. It's just more stuff to sift through when trying to understand this code.